Libraries/NativeAnimation/Nodes/RCTInterpolationAnimatedNode.m

```diff
@@ -29,6 +31,17 @@ - (instancetype)initWithTag:(NSNumber *)tag
       }
     }
     _outputRange = [outputRange copy];
+
+    if ([config[@"extrapolate"] isEqualToString:@"clamp"]) {
```

Comment:
JS will only send "extrapolateLeft" and "extrapolateRight" so no need to handle this case.